src/atomate2/common/schemas/math.py

```diff
@@ -0,0 +1,18 @@
+"""Schemas for matrices and vectors."""
+
+from typing import Tuple
+
+__all__ = ["Vector3D", "Vector6D", "Matrix3D", "MatrixVoigt"]
+
+
+Vector3D = Tuple[float, float, float]
```

Comment:
These are already in emmet: https://github.com/materialsproject/emmet/blob/main/emmet-core/emmet/core/math.py